agent/battle/execution/executor.py

```
@@ -0,0 +1,91 @@
+        self._click(coords.MASTER_SKILL_MENU_BTN)
+        time.sleep(_MASTER_SKILL_MENU_DELAY_S)
+        self._click(coords.MASTER_SKILL_CLICK[skill_index])
+        return True
+
+    def close_skill_use_dialog(self) -> bool:
+        """关闭点击 CD 技能后出现的"技能使用"提示窗。"""
+        self._click(coords.SKILL_USE_DIALOG_CLOSE_BTN)
+        return True
+
+    def tap_top_right_close(self) -> None:
+        """点击右上角关闭按钮，用于等待时持续点击以关闭可能弹出的遮挡层。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)